File: BackEnd/postman/HojaTrabajoSocial.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='HojaTrabajoSocial'
data={
  'accion':'crear',
  'descripcion': None,
  'idCentroMedico': None,
  'idExpediente': None,
}
print postear(data,url)
print

url='HojaTrabajoSocial'
data={
  'accion':'listarPorPaciente',
  'idExpediente': None,
}
print postear(data,url)
print

url='HojaTrabajoSocial'
data={
  'accion':'listarTodos',
  'idCentroMedico': None,
}
print postear(data,url)
print

url='HojaTrabajoSocial'
data={
  'accion':'actualizar',
  'descripcion': None,
  'idExpediente': None,
  'idCentroMedico': None,
  'idTS': None,
}
print postear(data,url)
print

File: BackEnd/ajax/HojaTrabajoSocial.js
$.ajax({
  url:CONST_SITIO_URL+'/services/HojaTrabajoSocial.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'descripcion': null,
    'idCentroMedico': null,
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/HojaTrabajoSocial.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/HojaTrabajoSocial.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTodos',
    'idCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/HojaTrabajoSocial.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'descripcion': null,
    'idExpediente': null,
    'idCentroMedico': null,
    'idTS': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


